### ISSUE TITLE
GTK 3.18 :: Totem :: Ubuntu 16.04 should have special videoplayer style

### ISSUE THREAD

**Issue by ingria** [numixproject/numix-gtk-theme#500]
Here's how Video's (Totem) media playback control panel looks with Numix theme:

![numix](https://cloud.githubusercontent.com/assets/2178368/17006914/655cc0d8-4ef5-11e6-86e7-5a1d2e1eab40.png)

Ambiance:

![ambiance](https://cloud.githubusercontent.com/assets/2178368/17006922/6f1f92bc-4ef5-11e6-9755-6caca0625c6f.png)

The panel has to have greater padding, also buttons should not have borders and background.

**Comment by lots0logs** [comment]
I agree that the style for the playback control panel  could use some love. However, the goal is not to look the same as Ambiance. The goal is for it to look like it fits in with the rest of the numix theme. With that in mind, only a few minor touchups are needed IMO.

**Comment by ingria** [comment]
> the goal is not to look the same as Ambiance

Of course. Ambiance just shows the main concept, created by Ubuntu devs (touch-friendly controls, not just usual buttons).
